docs/developer/extending/apps/synchronous-webhooks/transaction-webhooks.mdx

```diff
@@ -0,0 +1,432 @@
+---
+title: Transaction webhooks
+---
+
+# Introduction
+
+:::caution
+This feature is currently in **Feature Preview**. This part of Saleor is not complete
+and subject to change but is available to experiment and provide feedback.
+:::
+
+Transaction webhooks are synchronous webhooks that allow delegating the action for transaction to Saleor App.
+Synchronous means that these webhooks expect a response of a particular
+shape to be returned from the App to continue processing in Saleor. Transaction webhooks only
+support the HTTP(S) protocol; they are sent as POST requests with the application/json body
+and expect a response of the same content type.
```

Comment:
```suggestion
Transaction webhooks are synchronous webhooks that allow delegating the transaction to the Saleor App.
Synchronous means that these webhooks expect a response of a particular shape to be returned from the App to continue processing in Saleor. Transaction webhooks only support the HTTP(S) protocol; they are sent as POST requests with the `application/json` body and expect the response of the same content type.
```

Reply:
> Transaction webhooks are synchronous webhooks that allow delegating the transaction to the Saleor App.


It is not exactly delegating the transaction to Saleor app. The transaction is on our side, we trigger the request for an action to the App, and the app processes the action request and reports the event related to the transaction.